scripts/verify.bash

```diff
@@ -2,18 +2,20 @@
 # Copyright 2014 Canonical Ltd.
 # Licensed under the AGPLv3, see LICENCE file for details.
 
-# This is called from pre-push.bash to do some verification checks on 
+# This is called from pre-push.bash to do some verification checks on
 # the Go code.  The script will exit non-zero if any of these tests
 # fail. However if environment variable IGNORE_VET_WARNINGS is a non-zero
 # length string, go vet warnings will not exit non-zero.
 
-set -e 
+set -e
 
 VERSION=`go version | awk '{print $3}'`
 echo "go version $VERSION"
 
+FILES=`find * -name '*.go' -not -name '.#*' | grep -v vendor/`
```

Comment:
would it be more efficient to keep this inline below rather than pulling out to a separate var?

Reply:
it's used in two places, so I tried to put here.